### ISSUE TITLE
qemu-img missing parameter for 'backing format'?

### ISSUE THREAD

**Issue by fwilhe2** [afbjorklund/lima-the-hard-way#1]
I've tried to follow along and got this (running on fedora 40 with all required software installed from the repos):

```
lima-the-hard-way (main)$ qemu-img create -f qcow2 -b base.img disk.img
qemu-img resize disk.img 20g
qemu-img: disk.img: Backing file specified without backing format
Detected format of qcow2.
qemu-img: Could not open 'disk.img': Could not open 'disk.img': No such file or directory
```

at least for me, specifying `-F qcow2` has helped, not sure how universal that is

```
qemu-img create -F qcow2 -f qcow2 -b base.img disk.img
```

**Comment by afbjorklund** [comment]
Hmm, the base.img was _supposed_ to be a qcow2 - I don't think I needed to specify the source image format

EDIT: It was a new feature in QEMU 5.1 [https://www.qemu.org/docs/master/about/deprecated.html](https://www.qemu.org/docs/master/about/deprecated.html#qemu-img-backing-file-without-format-since-5-1)

**Comment by afbjorklund** [comment]
Fixed, 3f737eb156a9117f8a95bc80ba3814462ea1d500
